shared/charts/ssl-cert-check/values.yaml

```diff
@@ -0,0 +1,22 @@
+image:
+  repository: gpii/ssl-cert-check
+  tag: 0.0.1-gpii.0
+  pullPolicy: IfNotPresent
+
+## Set project_id for Stackdriver target project
+project_id: fake-project
```

Comment:
It might be preferable to not set this variable, and fail fast instead (when helm chart is deployed), instead of deploying chart with non-functional configuration.

Reply:
Ok, good argument, let me comment this out.